FORMAT.md

```diff
@@ -108,7 +111,7 @@ In addition to the default GeoJSON members, each feature may specify some additi
 |`type`|string|the type of the feature - this should match one of the strings in “Feature Types,” below.
 |`highlight`|boolean|whether to generate a highlight for the feature. Optional. Defaults to false.
 
-###### Attributes for placeholders 
+#### Attributes for placeholders 
```

Comment:
This section needs to be removed.  Placeholders were an early (abortive) attempt at the functionality now provided by the asset tool / entity sets.  NB.  There must be some support in eegeo-mobile for rendering props at particular scales, it seems 🧐